to_agol.py

```diff
@@ -0,0 +1,50 @@
+import arcgis
+import rasterio
+from rasterio.features import shapes
+from shapely.geometry import Polygon, shape
+
+
+@classmethod
+def from_shapely(cls, shapely_geometry):
+    return cls(shapely_geometry.__geo_interface__)
+
+arcgis.geometry.BaseGeometry.from_shapely = from_shapely
```

Comment:
Why is this being overwritten? A comment would be nice.

Reply:
Commented. It's to be able to grab the shape from a shapely object. Currently arcgis does not have a method to accomplish that.